internal/services/repository/resource_repository_docker_group_test.go

```diff
@@ -16,6 +16,7 @@ import (
 )
 
 func testAccResourceRepositoryDockerGroup() repository.DockerGroupRepository {
+	subdomain := fmt.Sprintf("test-repo-%s", acctest.RandString(10))
```

Comment:
please only add subdomain if skip pro test is false
pipeline is failing without pro license